components/calendar/src/iso.rs

```diff
@@ -395,6 +395,7 @@ impl Iso {
     }
 
     fn fixed_from_iso_integers(year: i32, month: i32, day: i32) -> i32 {
+        #[allow(clippy::unwrap_used)] // TODO(#1668) Clippy exceptions need docs or fixing.
```

Comment:
Praise: I like having an issue number `#1668` because we can close the issue once all of the TODOs in source code are gone.